extensions/amp-story/1.0/_locales/de.js

```diff
@@ -28,19 +28,19 @@ export default {
     string: 'Ablehnen',
   },
   [LocalizedStringId.AMP_STORY_DOMAIN_DIALOG_HEADING_LABEL]: {
-    string: 'Sehen Sie auf der ursprünglichen Domäne',
+    string: 'Auf der originalen Webseite anschauen',
```

Comment:
How about "Auf der ursprünglichen Domain anschauen"? 

"Webseite" has a slightly different meaning and in my opinion, "original" has a slightly different meaning in German compared to English.

Reply:
This is more gut feeling, but "ursprüngliche Domain" sounds wrong to me. This sounds like "natural/untouched/ Domain" to me...and I thought "original" would fit better.

But maybe something like "Auf der Ursprungsdomain anschauen"?

Reply:
I don't feel strongly between "original" and "ursprünglich". My thought on "original" was that it sounds like the current domain is a modified copy or something. While in fact it's just proxied. So "original domain" refers to the origin of the document rather than something being replicated.

I'm happy either way, but maybe avoid the compound word?